assets/src/edit-story/karma/fixture/containers/rightClickMenu.js

```diff
@@ -49,15 +50,53 @@ export class RightClickMenu extends Container {
     });
   }
 
+  // foreground media actions
+  get sendBackward() {
+    return this.getByRole('button', {
+      name: `Send backward, or use Option ${commandOrControl} Open Bracket on a keyboard`,
```

Comment:
Cruious, where are these strings assembled in the editor?

Reply:
These are created in the context menu's `MenuItem`: https://github.com/google/web-stories-wp/blob/main/packages/design-system/src/components/contextMenu/menuItem.js#L81